nix/pkgs/status-go/default.nix

```diff
@@ -0,0 +1,40 @@
+{ self, pkgs }:
+
+let
+  inherit (pkgs.lib) attrValues mapAttrs;
+
+  # Metadata common to all builds of status-go
+  meta = {
+    description = "The Status Go module that consumes go-ethereum.";
+    license = pkgs.lib.licenses.mpl20;
+    platforms = with pkgs.lib.platforms; linux ++ darwin;
+  };
+
+  version = "v10.27.0";
```

Comment:
This doesn't look right. We can't just hardcode version here. If we are forced to do so we might need something visible like a `VERSION` file at repo root.

We used to have a `VERSION` file but that was removed in favor of using `git describe`:
https://github.com/status-im/status-go/blob/076190e0af010fbbfc241b961bb313c408b11fb9/_assets/scripts/version.sh#L2

But we should be able to use commit as version, since `self` should contain it under `rev` attribute.